security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -0,0 +1,846 @@
+
+[MITRE ATT&CK Matrix For Kubernetes](https://www.darkreading.com/threat-intelligence/microsofts-kubernetes-threat-matrix-heres-whats-missing/a/d-id/1339106)
+
+### Acknowledgements
+
+This white paper is a community effort driven by the members of the CNCF Security-SIG. Thank you to everyone for their outstanding contributions. Special thanks to Emily Fox and Jeyappragash JJ.
+
+Contributors:
+
+- Aradhna Chetal - [TIAA](http://tiaa.org/)
+- Brandon Lum - [IBM](https://www.ibm.com/)
+- Chase Pettet - [Mirantis](https://www.mirantis.com/) (Chase.Pettet@mirantis.com)
+- Emily Fox - [US National Security Agency (NSA)](https://www.nsa.gov/)
+- Gadi Naor - [Alcide](http://www.alcide.io/)
+- Harmeet Singh - [IBM](https://www.ibm.com/)
+- Jeff Lombardo - Independent
+- Jeyappragash JJ - [Tetrate IO](https://www.tetrate.io/)
+- Pushkar Joglekar - [Visa](https://visa.com/)
+- Rowan Baker - [ControlPlane](https://control-plane.io/)
+- Andrew Martin - [ControlPlane](https://control-plane.io/)
+- Trishank Karthik Kuppusamy - [Datadog](https://www.datadoghq.com/)
+- Vinay Venkataraghavan -[Prisma Cloud (Palo Alto Networks)](https://www.paloaltonetworks.com/prisma/cloud)
+- Wayne Haber - [GitLab](https://about.gitlab.com/)
+- Mark Bower
+- Alex Chircop - StorageOS
+
+Reviewers:
+
+- @justincappos
+- @lumjjb
+- @whaber
+- @craigbox
+- @anvega
+- @magnologan
+- Alok Raj - [XenonStack](https://www.xenonstack.com/) ([alok@xenonstack.com](mailto:alok@xenonstack.com))
+- @nyrahul
+- @ranio1
+
+[1](#sdfootnote1anc) Another model to consider is Cloud, Clusters, Containers, and Code: [https://kubernetes.io/docs/concepts/security/overview/](https://kubernetes.io/docs/concepts/security/overview/)
+
+[2](#sdfootnote2anc)[Shifting security left](https://www.devsecops.org/blog/2016/5/20/-security) often leaves organizations to lapse operational security monitoring. It is important that security exists in all parts of the lifecycle and organizations continually evaluate other aspects of their business and technology processes where they may reach beyond modern security paradigms to embrace security as a culture and habit.
+
+[3] Example - [MITRE ATT&CK Framework for Kubernetes](https://www.darkreading.com/threat-intelligence/microsofts-kubernetes-threat-matrix-heres-whats-missing/a/d-id/1339106)
+
+[3](#sdfootnote3anc) Human capital is a vital asset necessary to the success of any organization, the corresponding intellectual property and relational capital brought as a result is equally in need of protection.
+
+[4](#sdfootnote4anc) https://blog.aquasec.com/malicious-container-image-docker-container-host
+
+[5](#sdfootnote5anc) According to Applied Software Measurement, Capers Jones, 1996 and adjusting for inflation - 85% of defects are introduced during coding with a cost of $41 to fix compared to a post release fix cost of $26,542.
+
+[6](#sdfootnote6anc) CIsecurity.org maintains a listing of benchmarks for hardening
```

Comment:
This should be cisecurity.org without upper case as it does _not_ represent CI (Continuous Integration).